### ISSUE TITLE
Disable Auto-phase creation for all project-types

### ISSUE THREAD

**Issue by RishiRajSahu** [appirio-tech/connect-app#4244]
There are some project types for eg. TaaS which. have predefined set of phases to be created on project creation, with the latest release of ability to add generic phase and customisable milestones it would be good to have that auto phase creation disabled

fyi @acshields @vikasrohit @maxceem

**Comment by maxceem** [comment]
IMHO, I would suggest keep this functionality in code (don't disable) and just update the Project Templates as we can decide whether to create a phase or no using metadata editor.

**Comment by RishiRajSahu** [comment]
@maxceem yep I meant disabling via template only :) . But you brought good point , so other than refactoring effort what else could be side-effect of removing from code, if we do it later in future sometime ?

**Comment by maxceem** [comment]
> But you brought good point , so other than refactoring effort what else could be side-effect of removing from code, if we do it later in future sometime ?

Yes. As we already have this implemented, if need to create some projects with predefined phases we can easily do that in the future.

**Comment by acshields** [comment]
@RishiRajSahu @maxceem - can this be wrapped up by 1/15?

**Comment by RishiRajSahu** [comment]
@acshields yes, just wanted your confirmation on disabling it for all project intakes ?

**Comment by acshields** [comment]
@RishiRajSahu - confirmed.

@maxceem - I know this won't make the 1/15 cut-off. One question for you: can this change be tackled purely by community effort, or will your/Rishi's direct involvement be required? I am asking because we did get approval to continue to run fixes during the next 6 weeks (when we will be doing our focused projects) if the work can be completely accomplished without direct Topcoder engineering or heavy Copilot support.

**Comment by maxceem** [comment]
@acshields I think this cannot be done purely by a community member. This task had 2 solutions, one way would involve Rishi efforts to update Project Templates or another way would involve some control from copilot.

**Comment by acshields** [comment]
@maxceem - Alright. I am going to deprioritize this one for now and save it for after 2/26 (our 6 week project cycle that kicks off next week), or until we finish the project work early (wishful thinking lol). Thanks for your input.

**Comment by RishiRajSahu** [comment]
@acshields Its done and updated for both Prod and Dev env.
